packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.stories.tsx

```diff
@@ -39,6 +39,6 @@ export const Usage = () => {
 const styles = {
   storybook: cssObj({
     margin: '20px',
-    width: '386px',
+    width: '95%',
```

Comment:
maybe $sm to use same size of Transactions screen
```suggestion
    width: '$sm',
```